packages/docusaurus/src/commands/swizzle/prompts.ts

```diff
@@ -29,19 +29,33 @@ export async function askThemeName(themeNames: string[]): Promise<string> {
 export async function askComponentName(
   themeComponents: ThemeComponents,
 ): Promise<string> {
-  const componentNames = themeComponents.allComponents;
+  function compareComponent(a: string, b: string) {
```

Comment:
😅  for node code I'd rather use lodash, it's much less verbose and clearer

Reply:
Ah, I don't know much about lodash sorting, feel free to refactor it :P